tests/test_type_inference/test_classdef.py

```diff
@@ -110,6 +111,7 @@ def test_builtin_method_call_bad_argument():
     """
     program = f'x = 1\n' \
               f'x.extend(1)\n'
+    raise SkipTest("TypeStore does not contain extend method")
```

Comment:
On a second look, it appears that this test isn't checking what it wants to. Please modify the test code to `x = [1, 2, 3]`. There should still be a `TypeFail`, but it should be a different one that the previous test. Maybe this test will actually pass.

Reply:
Decided to deal with these in a separate PR #410